### PR TITLE
Replace Benchmark.js with tinybench

### PR DESCRIPTION
**Pull request by yhz**

### CODE REVIEW

**Comment by ai** [comment]
Can you rebase your changes to the latest `main`? I like the idea of not hard-code the longest name.

File: test/benchmark.js [hunk 1]
```diff
@@ -70,14 +63,26 @@ suite
...
     rndm(21)
   })
-  .on('cycle', event => {
-    let name = event.target.name.padEnd('async secure-random-string'.length)
-    let hz = formatNumber(event.target.hz.toFixed(0)).padStart(10)
-    if (event.target.name === 'uid') {
-      name = '\nNon-secure:\n' + name
+
+const longestNameLength = bench.tasks.reduce((maxTaskNameLength, task) => {
+  task.addEventListener('cycle', () => {
```
Comment (ai) [inline test/benchmark.js:68]: Why do we call `task.addEventListener('cycle'` inside the cycle?

Why not after the `bench.tasks.reduce`?

Comment (yhz) [reply test/benchmark.js:68]: You're right, there was no reason to do it that way. I will make a pull request with the changes without hardcoding.

File: README.id-ID.md [hunk 2]
```diff
@@ -67,22 +67,22 @@ Ada dua buah perbedaan antara Nano ID dan UUID v4:
 
 ```rust
 $ node ./test/benchmark.js
-crypto.randomUUID         28,398,328 ops/sec
```
Comment (ai) [inline README.id-ID.md:70]: Something wrong happened with rebasing. Seems like it is easier to create another PR.

Or give me your email on GitHub and I will copy your changges manually.

File: test/benchmark.js [hunk 3]
```diff
@@ -70,14 +64,24 @@ suite
...
-    if (event.target.name === 'uid') {
-      name = '\nNon-secure:\n' + name
-    }
-    process.stdout.write(
-      `${name}${styleText('bold', hz)}${styleText('dim', ' ops/sec')}\n`
-    )
-  })
-  .run()
+
+const longestNameLength = bench.tasks.reduce((maxLength, task) => Math.max(maxLength, task.name.length), 0)
```
Comment (ai) [inline test/benchmark.js:68]: This line is not following the project’s code style.

`const` → `let` and be sure that it is follows 80 symbols limit.

**Pull request closed without merging**